dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
the problem here with this file is that they are using multiple define statements and that's not allowed so to over come that i did what i did (it's all explained) so the thing is here you can't rely on amd because you can't define a module then require it at some point later in that file so the only solution is to make hammer publicly availabe (from that file) so when hammer runs it's code it thinks it's being used normally but it's actually used with amd, i haven't tested what i told you i wanted to test, i am busy today so maybe tommorow or after it (i don't know when i will have free time) so yeah , just wait until i fully test and i will tell you the whole story ok ?